### ISSUE TITLE
Create a single repository for numerical methods in Fortran

### ISSUE THREAD

**Issue by Beliavsky** [NirupamKhanal/SimpsonsRule#1]
I suggest that you create a single repository perhaps named **NumericalMethodsInFortran** and put your Fortran codes there, instead of creating a separate repository for each algorithm that you code in Fortran.

**Comment by NirupamKhanal** [comment]
> I suggest that you create a single repository perhaps named **NumericalMethodsInFortran** and put your Fortran codes there, instead of creating a separate repository for each algorithm that you code in Fortran.

You are right. I have been meaning to do that but did not get around to it. I'll do it now.
